javascript/index.js

```diff
@@ -14,52 +14,91 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
-}
 
+  printMinutes();
+  printSeconds();
+
+}
 function printMinutes() {
-  // ... your code goes here
+  const mins = chronometer.twoDigitsNumber(chronometer.getMinutes());
+    minDec.innerText = mins[0];
+    minUni.innerHTML = mins[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  const secs = chronometer.twoDigitsNumber(chronometer.getSeconds());
+  secDec.innerText = secs[0];
+  secUni.innerHTML = secs[1];
 }
 
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
 }
 
-function printSplit() {
-  // ... your code goes here
-}
+  function printSplit() {    
+    let splitsCount = document.getElementById('splits')  
+    newSplit = document.createElement('li')  
+    newSplit.innerHTML = chronometer.splitClick()  
+    splitsCount.appendChild(newSplit)}
+
 
 function clearSplits() {
-  // ... your code goes here
+  document.getElementById("splits").innerHTML = "";
 }
 
 function setStopBtn() {
-  // ... your code goes here
```

Comment:
The logic defined line 51-53 is for setting the "start" button, therefor you should either rename your function or switch the logic with function setStartBtn